pkg/proxy/iptables/proxier.go

```diff
@@ -1063,6 +1063,10 @@ func (proxier *Proxier) syncProxyRules() {
 			}
 
 			if hasEndpoints {
+				destChain := svcChain
+				if utilfeature.DefaultFeatureGate.Enabled(features.ExternalPolicyForExternalIP) {
```

Comment:
On line 1078 we're still jumping to KubeMarkMasqChain even though it can be node local now, is that expected?

Reply:
Good catch. Will skip it then.